src/main/java/duke/command/CalendarCommand.java

```diff
@@ -0,0 +1,65 @@
+package duke.command;
+
+import duke.exception.UnknownDateTimeException;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CalendarCommand implements Command {
+    /**
```

Comment:
It would be good to have JavaDocs for your public classes as well, not just the methods, so that readers can understand what your class is for.

Reply:
Understood on that! Will integrate them into practice later on, but will still focus on what the `checkstyle.xml` file demands first to ensure regular workload completion of the iP.